Ok RHEL 8 CI doesn't like my __xstat fix:
```
CC       _libstat_wrapper.lo
_libstat_wrapper.c:13:12: error: static declaration of ‘__xstat’ follows non-static declaration
static int __xstat(int __ver, const char *__filename, struct stat *__stat_buf)
In file included from _libstat_wrapper.c:8:
 /usr/include/sys/stat.h:397:12: note: previous declaration of ‘__xstat’ was here
extern int __xstat (int __ver, const char *__filename,
```
Rethinking...

--- a/.gitignore
+++ b/.gitignore
@@ -61,3 +61,6 @@ test-driver
 /tests/functional/**/log_interlib_client
 /tests/functional/_pkgs
 /tests/functional/_results
+
+# blackbox data dumps from tests/examples
+*.fdata

--- a/examples/simplelog.c
+++ b/examples/simplelog.c
@@ -248,8 +248,8 @@ main(int32_t argc, char *argv[])
 	}
 
 	if (do_blackbox) {
-		logfile = NULL;
-		logfile[5] = 'a';
+		/* Emulate a bug to exercise the signal handler */
+		*(volatile int *)0 = 1;
 	} else {
 		qb_log_fini();
 	}

--- a/tests/check_map.c
+++ b/tests/check_map.c
@@ -500,7 +500,7 @@ test_map_traverse_ordered(qb_map_t *m)
 	for (i = 0; chars[i]; i++) {
 		qb_map_put(m, chars[i], chars[i]);
 	}
-	result = calloc(sizeof(char), 26 * 2 + 10 + 1);
+	result = calloc(26 * 2 + 10 + 1, sizeof(char));
 
 	i = 0;
 	for (p = qb_map_iter_next(it, &data); p; p = qb_map_iter_next(it, &data)) {

--- a/tests/crash_test_dummy.c
+++ b/tests/crash_test_dummy.c
@@ -72,7 +72,6 @@ sigsegv_handler(int sig)
 int32_t
 main(int32_t argc, char *argv[])
 {
-	char *logfile;
 	int i;
 
 	signal(SIGSEGV, sigsegv_handler);
@@ -100,9 +99,7 @@ main(int32_t argc, char *argv[])
 		func_two();
 	}
 
-	/* on purpose crash to make a blackbox.
-	 */
-       	logfile = NULL;
-	logfile[5] = 'a';
+	/* on purpose crash to make a blackbox. */
+	*(volatile int *)0 = 1;
 	return 0;
 }

